app/Http/Requests/UpdatePassword.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdatePassword extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'current_password' => 'required',
+            'new_password' => 'required|alpha_dash|max:255',
```

Comment:
As @UncertaintyP suggested, we should not be using the `alpha_dash` validation rule which I agree, however what ***can*** we do to strengthen this validation to what is expected from the user input ?